nemo_curator/stages/video/io/clip_writer.py

```diff
@@ -334,33 +334,6 @@ def _write_clip_mp4(self, clip: Clip, *, filtered: bool = False) -> ClipStats:
             clip_stats.num_passed += 1
         return clip_stats
 
-    def _write_clip_embedding_to_buffer(self, clip: Clip) -> ClipStats:
-        clip_stats = ClipStats()
-        if clip.intern_video_2_embedding is not None:
-            self._iv2_embedding_buffer.append(
-                {
-                    "id": str(clip.uuid),
-                    "embedding": clip.intern_video_2_embedding.reshape(-1).tolist(),
-                },
-            )
-        elif self.generate_embeddings and self.embedding_algorithm == "internvideo2":
-            logger.error(
-                f"Clip {clip.uuid} from {clip.source_video} has no InternVideo2 embedding, skip adding to buffer"
-            )
-        if clip.cosmos_embed1_embedding is not None:
-            self._ce1_embedding_buffer.append(
-                {
-                    "id": str(clip.uuid),
-                    "embedding": clip.cosmos_embed1_embedding.reshape(-1).tolist(),
-                },
-            )
-        elif self.generate_embeddings and self.embedding_algorithm == "cosmos-embed1":
-            logger.error(
-                f"Clip {clip.uuid} from {clip.source_video} has no Cosmos-Embed1 embedding, skip adding to buffer"
-            )
-
-        return clip_stats
-
```

Comment:
There were two identical `_write_clip_embedding_to_buffer` methods here (my guess is it's was a copy/paste issue at some point)